src/hotspot/cpu/x86/stubGenerator_x86_32.cpp

```diff
@@ -4004,6 +4028,10 @@ class StubGenerator: public StubCodeGenerator {
     StubRoutines::x86::_vector_int_mask_cmp_bits = generate_vector_mask("vector_int_mask_cmp_bits", 0x00000001);
     StubRoutines::x86::_vector_iota_indices = generate_iota_indices("iota_indices");
 
+    if (VM_Version::supports_avx2() && UsePopCountInstruction) {
```

Comment:
Do we need it if VM_Version::supports_avx512_vpopcntdq()?

Reply:
DONE